core-primitives/stf-executor/src/executor_tests.rs

```diff
@@ -263,6 +263,8 @@ pub(crate) fn init_state_and_shard_with_state_handler<S: HandleState<StateT = St
 	let _hash = state_handler.initialize_shard(shard).unwrap();
 
 	let (lock, mut state) = state_handler.load_for_mutation(&shard).unwrap();
+
+	#[cfg(feature = "test")]
```

Comment:
Isn't the whole file only used in tests anyhow?

Reply:
true. seems I was seeking a problem in the wrong spot